### DISCUSSION TITLE
Support for bincode crate for encoding/decoding

### DISCUSSION THREAD

**Discussion by dmbfm** [bitshifter/glam-rs#544] [Ideas]
I'm interesting in implementing this feature since I need it for a project I am working on where I use the [bincode crate](https://github.com/bincode-org/bincode) for encoding/decoding binary streams. Is there interest for this feature? 

Update: I already have working code for this, with the caveat that bincode is currently moving from v1 to v2, and my code works for the  latest v2 release candidate.

**Comment by bitshifter** [comment]
If you have something working feel free to make a draft PR so I can take a look at it. Adding optional dependencies is a bit of a conundrum as there are a lot of different crates that people might want to use with glam that are of varying levels of popularity and maturity, I don't want to end up adding a feature for a crate that ends up being abandoned or there's only one person who wants to use it. bincode itself at least looks pretty well established.

**Reply by dmbfm** [reply]
Yeah, that's why I wanted to see if there is any demand from this. Because if it's not really necessary for other users, it's not worth it to add this. My use case is somewhat tangential and I may not need this in the future, so it only makes sense if there is some demand.

**Comment by bitshifter** [comment]
closing as bincode has been archived.
